src/VisualStudio/Core/Def/Packaging/Patching/Delta.cs

```diff
@@ -0,0 +1,248 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.LanguageServices.Packaging.Patching
+{
+    internal static unsafe class Delta
```

Comment:
Where did this come from? Do we need to keep it in sync with some code elsewhere? Should there be a comment to not ever touch this or something?